bokehjs/src/lib/models/annotations/legend.ts

```diff
@@ -520,6 +543,8 @@ export class Legend extends Annotation {
       border_line_width: 1,
       background_fill_color: "#ffffff",
       background_fill_alpha: 0.95,
+      item_background_fill_color: "#f1f1f1",
+      item_background_fill_alpha: 0.8,
```

Comment:
I think discussion is needed before turning this on *by default*. And if we turn do turn it on by default, are there other properties to change simultaneously? (For instance: I think the alternation turned on with the current fairly sizable border looks a bit clunky, or maybe something more subtle like `#fafafa` would help). 

There is also a matter of updating more than a few gallery thumbnails. 

cc @bokeh/core

Reply:
Alternatively, we could avoid making decisions about things like this in PR, and always leave new visual changes off by default in the PR, and then add a link to a running issue for the release branch noting that there is a potential default change related to the PR. That way all default changes can be discussed in one place, and single PR can be made to update everything as decided (including, e.g. thumbnails at the same time). To me, that would help keep changes like this more organized.